packages/aws-cdk/test/api/test.bootstrap.ts

```diff
@@ -144,7 +144,57 @@ export = {
       region: 'us-east-1',
       name: 'mock',
     }, sdk, 'mockStack', undefined, {
-      kmsKeyId: 'myKmsKey',
+      kmsKeyId: 'myKmsKey', tags: undefined
+    });
+
+    // THEN
+    test.equals(ret.noOp, false);
+    test.equals(executed, true);
+
+    test.done();
+  },
+  async 'do bootstrap using tags'(test: Test) {
+    // GIVEN
+    const sdk = new MockSDK();
+
+    let executed = false;
+
+    sdk.stubCloudFormation({
+      describeStacks() {
+        return {
+          Stacks: []
+        };
+      },
+
+      createChangeSet(info: CreateChangeSetInput) {
+        const template = fromYAML(info.TemplateBody as string);
+        const bucketProperties = template.Resources.StagingBucket.Properties;
+        test.equals(bucketProperties.BucketName, undefined, 'Expected BucketName to be undefined');
+        test.equals(bucketProperties.BucketEncryption.ServerSideEncryptionConfiguration[0].ServerSideEncryptionByDefault.KMSMasterKeyID,
+            'myKmsKey', 'Expected KMSMasterKeyID to be myKmsKey');
```

Comment:
copy paste leftovers? don't think you intended to test this behaviour in this test